data-products/src/test/scala/org/sunbird/analytics/job/report/TestCourseMetricsJob.scala

```diff
@@ -148,50 +152,11 @@ class TestCourseMetricsJob extends BaseReportSpec with MockFactory {
     CourseMetricsJob.prepareReport(spark, storageConfig, reporterMock.loadData,config)
 
     implicit val batchReportEncoder: Encoder[BatchReportOutput] = Encoders.product[BatchReportOutput]
```

Comment:
All the assertions for the CourseMetrics job has been removed. I am not sure if it is ok to delete all the assertions. Can you please elaborate why we don't need these assertions?

Reply:
Added assertions